Dockerfile-api

```diff
@@ -17,4 +17,4 @@ ENV HOST 0.0.0.0
 ENV PORT 8000
 EXPOSE 8000
 
-CMD ["/app/server", "--http-host=0.0.0.0", "--http-port=8000", "--repositories", "keboola|https://github.com/keboola/keboola-as-code-templates.git|main"]
+CMD ["/app/server", "--http-host=0.0.0.0", "--http-port=8000", "--repositories", "keboola|https://github.com/keboola/keboola-as-code-templates.git|main;keboola|https://github.com/keboola/keboola-as-code-templates.git|beta;keboola|https://github.com/keboola/keboola-as-code-templates.git|dev"]
```

Comment:
Pridane dalsie repos do Dockerfile.

Reply:
To neva že se ty repozitáře jmenují stejně? 🤔

Reply:
:man_facepalming:  No ide to, to som skusal ... ale je pravda, ze ked som teraz skusal `repositories` endpoint, tak je tam 3x `keboola`.

Vo vacsine kodu pouzivame `Hash`, tam je `Ref`, takze tam problem nie je.
Ale ide o to, ze ktory je teraz default `Keboola`, ... ide to nahodou asi preto, lebo `main` je prvy, idem to fixnut, dik :+1: